packages/subgraph/tests/multisig/multisig.test.ts

```diff
@@ -386,36 +380,41 @@ test('Run Multisig (handleMembersRemoved) mappings with mock event', () => {
 
 test('Run Multisig (handleMultisigSettingsUpdated) mappings with mock event', () => {
   // create state
-  let entityID = Address.fromString(CONTRACT_ADDRESS).toHexString();
-  let multisigPlugin = new MultisigPlugin(entityID);
-  multisigPlugin.dao = DAO_ADDRESS;
-  multisigPlugin.pluginAddress = Bytes.fromHexString(CONTRACT_ADDRESS);
-  multisigPlugin.onlyListed = false;
-  multisigPlugin.save();
+  let entityID = createMultisigPluginState().id;
 
   // create event
+  let onlyListed = 'true';
```

Comment:
I'd still make it `let onlyListed =  true` instead of `string`. and in the `assert.fieldEquals`, I'd do `onlyListed.toString()`. The reasoning is what if you need  `onlyListed` checks in the tests ? doing `onlyListed === 'true'` is kind of not ideal. in the `assert` though, it doesn't make any difference. Up to you.

Reply:
amended via template sting